PF-Assignment4/1. TowerOfHanoi/TowerOfHanoi.java

```diff
@@ -0,0 +1,37 @@
+import java.util.ArrayList;
+import java.util.InputMismatchException;
+
+/**
+ * @author Preeti Kumawat
+ * Class Name: TowerOfHanoi
+ * Date: 18-07-2017
+ *
+ */
+public class TowerOfHanoi {
+	ArrayList<String> listOfMovements = new ArrayList<String>(); 
+	
+	/**
+	 * @param source
+	 * @param destination
+	 * @param auxiliary
+	 * @param numberOfDisk
+	 * @return ArrayList<String>
+	 */
+	ArrayList<String> towerOfHanoi(String source, String destination, String auxiliary, int numberOfDisk){
```

Comment:
Method name should not be the same as class name. It should be more logical 'getMoves'